frontend/store/auctions.ts

```diff
@@ -358,6 +359,11 @@ export const actions = {
             commit('setAreTakeEventsFetching', false);
         }
     },
+    async fetchSurplusAuctions({ rootGetters }: ActionContext<State, State>) {
+        const network = rootGetters['network/getMakerNetwork'];
+        const auctions = await fetchActiveSurplusAuctions(network);
+        return auctions;
+    },
```

Comment:
Let's create another `surplus` store instead and not mix collateral with surplus auctions